kura/org.eclipse.kura.wire.helper.provider/about_files/epl-v10.html

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```

Comment:
Not every. But still, because it is wrong there is no reason to continue doing so.